integration/default_test.go

```diff
@@ -155,8 +166,13 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 			)
 
 			image, logs, err = pack.WithNoColor().Build.
-				WithPullPolicy("never").
-				WithBuildpacks(buildpack, buildPlanBuildpack).
+				WithExtensions(
```

Comment:
Similar change to previous for consistenccy with other buildpack integration tests.